api/src/controllers/torrent.js

```diff
@@ -8,6 +8,7 @@ import User from "../schema/user";
 import Comment from "../schema/comment";
 import Group from "../schema/group";
 import { createGroup, addToGroup, removeFromGroup } from "./group";
+import contentDisposition from "content-disposition";
```

Comment:
Please move this external import above the local imports (I think eslint should've [warned](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md) about this) and then I will merge

Reply:
done it